contracts/upgradeable_contracts/arbitrary_message/ForeignAMB.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.4.24;
+
+import "./BasicForeignAMB.sol";
+
+contract ForeignAMB is BasicForeignAMB {
+    event UserRequestForAffirmation(bytes encodedData);
+    event RelayedMessage(address sender, address executor, bytes32 transactionHash, bool status);
+
+    function getMaxGasPerTx() internal returns (uint256) {
```

Comment:
Could we invoke `maxGasPerTx` directly instead of `getMaxGasPerTx` in that places where it is used?

Reply:
Yes, that makes sense. https://github.com/poanetwork/poa-bridge-contracts/pull/77/commits/e1eb585dd20c84e8686717424225c25bfc1ea3d7